test/language/module-code/import-attributes/allow-nlt-before-with.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2021 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: >
+  `with` AttributesKeyword in WithClause in ImportDeclaration can
+  be preceded by a line terminator
+esid: sec-modules
+info: |
+  ImportDeclaration:
+    import ModuleSpecifier[no LineTerminator here] WithClause;
+
+  WithClause:
+    AttributesKeyword {}
+    AttributesKeyword { WithEntries ,opt }
+
+  AttributesKeyword:
+    with
+    [no LineTerminator here] assert
+
+negative:
+  phase: resolution
+  type: SyntaxError
+features: [import-attributes, globalThis]
+flags: [module, raw]
+---*/
+
+$DONOTEVALUATE();
```

Comment:
Lint error:
```
/home/circleci/test262/test/language/module-code/import-attributes/allow-nlt-before-with.js: NEGATIVE - Negative tests of type "early" must include a `throw` statement
```

isn't `$DONOTEVALUATE()` enough?

Reply:
Not in "raw" files, presumably because they don't have access to `$DONOTEVALUATE`:
```suggestion
throw "Test262: This statement should not be evaluated.";
```